### ISSUE TITLE
Breaks on dot with Ruby 2.6.1

### ISSUE THREAD

**Issue by mhw** [samphippen/rubyfmt#41]
* Ruby version: 2.6.1
* Rubyfmt git sha: 37fee6060eaa283a758ac18d0de780ccf0345a96

## Input file

```ruby
$LOAD_PATH.unshift 'lib'
```

## Rubyfmt's output

```
Traceback (most recent call last):
        9: from /Users/mhw/bin/rubyfmt:2288:in `<main>'
        8: from /Users/mhw/bin/rubyfmt:2285:in `main'
        7: from /Users/mhw/bin/rubyfmt:2145:in `format_program'
        6: from /Users/mhw/bin/rubyfmt:2145:in `each'
        5: from /Users/mhw/bin/rubyfmt:2146:in `block in format_program'
        4: from /Users/mhw/bin/rubyfmt:2139:in `format_expression'
        3: from /Users/mhw/bin/rubyfmt:2079:in `block in format_expression'
        2: from /Users/mhw/bin/rubyfmt:1005:in `format_command_call'
        1: from /Users/mhw/bin/rubyfmt:141:in `with_start_of_line'
/Users/mhw/bin/rubyfmt:1007:in `block in format_command_call': got something other than a dot (RuntimeError)
```

## This breaks because

Should not crash, and instead output

```
$LOAD_PATH.unshift("lib")
```

**Comment by samphippen** [comment]
could you try 634f8e6c74247a8cd90d7ff6fdea9dc9ccb60e2b and tell me if it works?

**Comment by bquorning** [comment]
There’s a typo in your fix in 634f8e6c74247a8cd90d7ff6fdea9dc9ccb60e2b

 - Ruby version: `ruby 2.6.1p33 (2019-01-30 revision 66950) [x86_64-darwin18]`
 - Rubyfmt git sha: 634f8e6c74247a8cd90d7ff6fdea9dc9ccb60e2b

```
$ echo 'Foo.bar "baz"' | ruby src/rubyfmt.rb
Traceback (most recent call last):
	10: from src/rubyfmt.rb:2329:in `<main>'
	 9: from src/rubyfmt.rb:2326:in `main'
	 8: from src/rubyfmt.rb:2176:in `format_program'
	 7: from src/rubyfmt.rb:2176:in `each'
	 6: from src/rubyfmt.rb:2177:in `block in format_program'
	 5: from src/rubyfmt.rb:2170:in `format_expression'
	 4: from src/rubyfmt.rb:2109:in `block in format_expression'
	 3: from src/rubyfmt.rb:1009:in `format_command_call'
	 2: from src/rubyfmt.rb:141:in `with_start_of_line'
	 1: from src/rubyfmt.rb:1011:in `block in format_command_call'
src/rubyfmt.rb:1001:in `is_normal_dot': undefined local variable or method `dot' for main:Object (NameError)
Did you mean?  not
```

Fixing the typo:
```diff
diff --git a/src/rubyfmt.rb b/src/rubyfmt.rb
index 22a67d7..966a4ee 100755
--- a/src/rubyfmt.rb
+++ b/src/rubyfmt.rb
@@ -998,7 +998,7 @@ def format_symbol_literal(ps, literal)
 end

 def is_normal_dot(candidate)
-  candidate == :"." || (candidate.is_a?(Array) && dot[0] == :@period)
+  candidate == :"." || (candidate.is_a?(Array) && candidate[0] == :@period)
 end

 def format_command_call(ps, expression)
```

It now works fine:
```
$ echo 'Foo.bar "baz"' | ruby src/rubyfmt.rb
Foo.bar("baz")
```
